menstruation/handlers.py

```diff
@@ -17,6 +19,7 @@
 from menstruation.query import Query
 
 user_db = config.user_db
+TIME_PATTERN = r"([01][0-9]|2[0-3]|\s[0-9]):[0-5][0-9]"
```

Comment:
why `\s`?

Reply:
For example, if I left out the /s, the regex would recognize 44:33 as 4:33. I would like to avoid this and I assume that someone would enter something like "/subscribe 3€ 9:00".

Reply:
I'd do `([01]?[0-9]|2[0-3]):[0-5][0-9]`

Reply:
Okay, I'm gonna leave `\s` out.

```suggestion
TIME_PATTERN = r"([01][0-9]|2[0-3]|[0-9]):[0-5][0-9]"
```
because I dont know if I can cange the following line without breaking anything.

`filter_text = demojize("".join(context.args))`

Let's hope that nobody gets the idea to enter wrong times xD